source/_integrations/apprise.markdown

```diff
@@ -14,28 +14,48 @@ To use Apprise supported notifications, add the following to your `configuration
 ```yaml
 # Example configuration.yaml entry using URLs
 notify:
-  - platform: apprise
+  - name: NOTIFIER_NAME
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
Thank you for your fast review!  I added the name: because i was having a look at how the [Slack Documentation](https://www.home-assistant.io/integrations/slack/) looked.  Regardless, I dropped the name as requested! :slightly_smiling_face:

Reply:
Thanks! Yeah, we haven't been to strict in the past, so a lot to be done! 😉